frontend/src/components/endpoints/Details.tsx

```diff
@@ -1,7 +1,7 @@
 import { useTranslation } from 'react-i18next';
 import { useLocation, useParams } from 'react-router-dom';
 import { ResourceClasses } from '../../lib/k8s';
-import Endpoints, { KubeEndpoint } from '../../lib/k8s/endpoints';
+import Endpoint, { KubeEndpoint } from '../../lib/k8s/endpoints';
```

Comment:
I know it's confusing, but the K8s resource kind is indeed plural! https://kubernetes.io/docs/concepts/services-networking/service/#endpoints